src/nyc_trees/apps/home/views.py

```diff
@@ -51,10 +51,11 @@ def _global_counts(past_week=False):
     else:
         blocks_mapped = Survey.objects.all().distinct('blockface_id').count()
     if blocks_mapped > 0:
-        blocks_percent = "{:.0%}".format(
-            float(blocks_mapped) / float(blocks_total))
+        fraction_mapped = float(blocks_mapped) / float(blocks_total)
+        fmt = "{:.0%}" if fraction_mapped >= .1 else "{:.1%}"
+        blocks_percent = fmt.format(fraction_mapped)
     else:
-        blocks_percent = "0%"
+        blocks_percent = "0.0%"
```

Comment:
Minor, but I think we could safely remove this conditional to create a single code path.